modules/phase_field/doc/content/source/materials/ADMathFreeEnergy.md

```diff
@@ -0,0 +1,23 @@
+# ADMathFreeEnergy
```

Comment:
What does this name mean?!

Reply:
It means AD version of MathFreeEnergy :-D

Reply:
Either way, the reasoning behind the original name was that this is not a physical free energy, but just a simple mathematical model free energy.